modules/p2-profile-gen/pom.xml

```diff
@@ -961,854 +957,6 @@
                             <artifactRepository>file:${basedir}/target/p2-repo</artifactRepository>
                         </configuration>
                     </execution>
-
-
-                    <!-- Only IS Profile (IS Front-End + IS Back-End) -->
-                    <execution>
-                        <id>creating-is-profile</id>
```

Comment:
AFAIR, IS profile was not having the embedded workflow engine. That was the only change in IS profile compared to default profile. Have we already removed the embedded workflow engine from the default profile?

Reply:
Yes it was removed in https://github.com/wso2/product-is/pull/15363